hm-oauth-rest-v1.yml

```diff
@@ -27,6 +27,26 @@ paths:
             $ref: '#/definitions/AccessTokensRequest'
       operationId: ApiWeb.OAuthAccessTokenController.create
       description: Creates and renews AccessTokens.
+    delete:
+      tags:
+        - OAuth2
+      summary: ''
```

Comment:
Looks like summary is the "main description"!
https://swagger.io/docs/specification/basic-structure/

please change this and we will change this to:
```
      summary: 'Revokes AccessTokens'
      description: 'This API implements rfc7009#section-2'
```

Reply:
Done in ea32422087723a11de281565869220a5109f3494